src/v/cluster/security_manager.cc

```diff
@@ -108,6 +123,60 @@ do_apply(create_user_cmd cmd, security::credential_store& store) {
     return errc::success;
 }
 
+/*
+ * handle: rename role command
+ */
+static std::error_code
```

Comment:
nit: anonymous namespace instead of `static`.

Reply:
done